src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -550,26 +553,13 @@ ServerHandlerImp::processRequest (Port const& port,
         }
     }
 
-    // Parse id now so errors from here on will have the id
-    //
-    // VFALCO NOTE Except that "id" isn't included in the following errors.
-    //
-    Json::Value const& id = jsonRPC [jss::id];
+    /* ---------------------------------------------------------------------- */
+    // Determine role/usage so we can charge for invalid requests
+    Json::Value const& id = jsonRPC ["id"];
```

Comment:
Can't this line just be removed now?  And if not, why use `"id"` in place of `jss::id`?